app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -0,0 +1,154 @@
+<article>
+  <section class="statement">
+    <div>
+      <span class="pink xlarge bold">Teaching is&hellip;</span>
+    </div>
+
+    <div>
+      <span class="pink medium bold">
+        <ul class="slashed-list">
+          <li>generation-shaping</li>
+          <li>innovative</li>
+          <li>life-changing</li>
+        </ul>
+      </span>
+    </div>
+  </section>
+
+  <section>
+    <h2 class="heading-s">Find your future career</h2>
+
+    <p>
+      Finding out which career is right for you is not always easy. How do you know what something is like if you've never done it before?
+
+    <p>
+      Our explore teaching advisers help you discover whether the classroom is where you belong. You can get support and guidance from your dedicated adviser who can answer all your questions about what teaching is really like.
+    </p>
+
+    <p>
+      All our advisers are experienced former teachers who can help you gain an insight into the world of teaching. You can talk to your own adviser as little or as much as you'd like.
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+    heading: "Bring it",
+    colour: "pink",
+    image_1_path: "media/images/content/blog/abigail-beeley.jpg",
+    image_1_alt: "Maths teacher Abigail Beeley in front of a maths display board.",
+    image_2_path: "media/images/content/blog/giorgio-rubbo.jpg",
+    image_2_alt: "Biology teacher Giorgio Rubbo wearing a suit and tie.",
+  ) do %>
+
+    <p>
+      Every teacher brings something different to the classroom.
+    </p>
+
+    <p>
+    What could you bring?
+    </p>
+  <% end %>
+
+  <section>
+    <figure>
+      <blockquote class="pink-quotes">
+        <p class="bold">
+          I promise the level of satisfaction is like nothing you’ll get in another job. I wake up every morning and want to get into school.
+        </p>
+      </blockquote>
+
+      <figcaption>Phil, teacher</figcaption>
+    </figure>
+  </section>
+
+  <section class="columns">
+    <div>
+      <h2 class="heading-s">What explore teaching advisers can do</h2>
+
+      <p>
+        Your adviser can help you:
+      </p>
+    </div>
+
+    <div>
+      <ul class="badges">
+        <li class="badge">understand what teaching is really like</li>
+        <li class="badge">connect with other students</li>
+        <li class="badge">build your confidence and transferable skills</li>
+        <li class="badge">get classroom experience</li>
+        <li class="badge">understand the process to become a teacher</li>
+      </ul>
+    </div>
+  </section>
+
+  <section>
+    <h2 class="heading-s">How to get an explore teaching adviser</h2>
+
+    <p>
+        To be eligible for an explore teaching adviser, you just need:
+    </p>
+
+    <ul>
+      <li>
+        to be studying for an undergraduate degree, not in your final year of study
```

Comment:
```suggestion
        to be studying for an undergraduate degree and not in your final year of study
```
Should we say 'any' undergraduate degree (if that's correct) - to emphasise that it doesn't need to be about teaching? (Not sure about this!)